aeneas/src/v3/V3.v3

```diff
@@ -152,6 +152,14 @@ component V3 {
 			return buf;
 		}
 		match (val) {
+			x: Box<u1> => buf.putd(x.val);
```

Comment:
I've been trying to move the compiler away from using `Box<int>`, `Box<long>`, etc, to having dedicated value classes, instead of a parameterized class. I think it'd be less error prone to try to use `Box` to enforce the bitwidth of immediates. Therefore I think it'd be better to use `Box<int>` for now for all integer-sized immediates.